### PR TITLE
Authentication react

### PR DESCRIPTION
**Pull request by Varu98**

### CODE REVIEW

File: src/backend/db/users.js [hunk 1]
```diff
@@ -17,3 +17,18 @@ export const users = [
...
+//   password: "mistry",
+//   createdAt: formatDate(),
+//   updatedAt: formatDate(),
+// },
+// {
+// "firstName": "Adarsh",
+// "lastName": "Balika",
+// "email": "adarshbalika@gmail.com",
+// "password": "adarshbalika",
+// },
```
Comment (haarsh24) [inline src/backend/db/users.js:34]: remove comented code

File: src/frontend/Utilities/loginService.js [hunk 2]
```diff
@@ -18,9 +18,8 @@ export const loginService = (e, loginData, navigate, authDispatch) => {
       console.log(data, status);
       if (status === 200) navigate("/");
     } catch (error) {
-      console.log(error);
+      console.log("Login: ", error);
     }
   })();
-
   console.log(loginData);
```
Comment (haarsh24) [inline src/frontend/Utilities/loginService.js:26]: remove console log

File: src/frontend/Utilities/signupService.js [hunk 3]
```diff
@@ -17,7 +17,7 @@ export const signupService = (e, signupData, authDispatch, navigate) => {
       // navigate to somewhere...
       if (status === 201) navigate("/login");
     } catch (error) {
-      console.log(error);
+      console.log("Signup: ", error);
```
Comment (haarsh24) [inline src/frontend/Utilities/signupService.js:20]: same here

**Review by haarsh24 — APPROVED** [review]
clean code ,
good folder structure ,
 remove console logs and commented code

**Review by preethasri — COMMENTED** [review]
authentication working fine.good to merge

**Pull request merged**